lib/ftu/authentication.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:async';
+
```

Comment:
Any import statement that isn't from a local file can be grouped together! You can delete the empty line between them.

Reply:
Thanks!!